src/simple/Phone.js

```diff
@@ -113,14 +130,18 @@ class Phone extends Emitter {
     return this.phone && this.phone.turnCameraOn(callSession);
   }
 
-  startScreenSharing(constraints: Object) {
+  async startScreenSharing(constraints: Object) {
     return this.phone && this.phone.startScreenSharing(constraints);
   }
 
   stopScreenSharing() {
     return this.phone && this.phone.stopScreenSharing();
   }
 
+  sendDTMF(callSession: CallSession, tone: string) {
```

Comment:
We have to be more careful on arguments order. `sendMessage` has `message (content), sipSession (call)` but here it's `session (call), tone (content)`.

Reply:
Couldn't agree more. However, some wise man once advised me not to mess with the API in already place :). This said, it's very tempting to move `WebRTCPhone`'s `sipSession` to the second slot, since it's optional (and body isn't).

Reply:
Be careful don't break the API for older dev.

Reply:
Yes but `sendDTMF` is a new method in the simple SDK. There is no breaking change here.

Reply:
Oh -- didn't realize you were referring specifically to `sendDTMF` here
[picking up the dropped mic and starting to sing "Feelings"]